### PR TITLE
[MFTF] User Sees keywords in the image preview Automation + Hide button

### PR DESCRIPTION
**Pull request by engcom-Golf**
### Description (*)
User Sees keywords in the image preview Automation + Added button to hide keywords after view all.

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#376:User Sees keywords in the image preview Automation
2. ...

### Manual testing scenarios (*)
1. Run functional tests
2. Click View All and Hide on image preview

### CODE REVIEW

File: AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/image-preview.html [hunk 1]
```diff
@@ -8,13 +8,13 @@
 <div class="masonry-image-preview" if="$col.isVisible($row())" data-image-preview ko-style="$col.getStyles($row())">
     <div class="container">
         <div class="action-buttons">
-            <button class="action-previous" type="button" data-bind="click: function () { $col.prev($row()); }">
+            <button class="action-previous" type="button" data-bind="click: function () { $col.prev($row()); $col.hideAllKeywords($row()); }">
                 <span translate="'Previous'"/>
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/image-preview.html]: Hiding keywords on switching between images is a part of view layer logic, it should not be tiered to the template. Please extend prev, next and hide methods in the component.

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @engcom-Golf thanks for the pull request, please see my review comments

File: AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/image-preview.html (hunk 1 above)
Comment (Nazar65) [reply AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/image-preview.html]: Ok

**Comment by Nazar65** [comment]
Hi @sivaschenko  done :heavy_check_mark:

File: AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        </after>
+        <actionGroup ref="AdminAdobeStockOpenSearchModalActionGroup" stepKey="openSearchModal"/>
+        <click selector="{{AdobeStockSection.imagesThumbnails}}" stepKey="clickOnThumbnail"/>
+        <!-- Assert that image attributes keywords visible "-->
+        <seeElement selector="{{AdobeStockSection.imageKeywordsTitle}}" stepKey="seeAttributeTitle"/>
+        <actionGroup ref="AssertNumberOfKeywordsInImagePreviewActionGroup" stepKey="verifyKeywordsCount">
+            <argument name="keywordsNumber" value="5"/>
+        </actionGroup>
+        <!-- Click view all keywords -->
+        <click selector="{{AdobeStockSection.imagePreviewKeywordsButtons('View all')}}" stepKey="clickOnViewAlKeywords"/>
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml:36]: verificatioin of view all button functionality is missing, please check that button disappears and keywords number increases

File: AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminUserSeesKeywordsInTheImagePreviewTest">
+        <annotations>
+            <stories value="Admin User Sees keywords in the image preview "/>
+            <title value="Admin should be able to see keywords in the image preview"/>
+            <description value="Admin should be able to able to see keywords in the image preview"/>
+            <severity value="CRITICAL"/>
+            <group value="AdobeStockIntegration"/>
+        </annotations>
+        <before>
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml:19]: Please use the following before and after nodes:

        <before>
            <actionGroup ref="LoginActionGroup" stepKey="LoginAsAdmin"/>
            <actionGroup ref="AdminAdobeStockSetConfigActionGroup" stepKey="setCorrectModuleConfig"/>
        </before>
        <after>
            <actionGroup ref="resetAdminDataGridToDefaultView" stepKey="resetAdminDataGridToDefaultView"/>
            <actionGroup ref="logout" stepKey="logout"/>
        </after>

**Review by sivaschenko — CHANGES_REQUESTED** [review]

**Comment by sivaschenko** [comment]
@Nazar65 thanks for the updates, please take a look at a couple of additional comments

File: AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml (hunk 2 above)
Comment (engcom-Golf) [reply AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml:36]: ok

File: AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml (hunk 3 above)
Comment (engcom-Golf) [reply AdobeStockImageAdminUi/Test/Mftf/Test/AdminUserSeesKeywordsInTheImagePreviewTest.xml:19]: ok

**Comment by engcom-Golf** [comment]
Hi @sivaschenko could please review new changes ?

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**